src/features/user/CompleteSignup/index.tsx

```diff
@@ -0,0 +1,442 @@
+          bodyToSend = {
+            type: 'education',
+            firstname: firstName,
+            lastname: lastName,
+            name: nameOfOrg,
+            country: country,
+            mayPublish: !isPrivateAccount,
+            mayContact: isSubscribed,
+            oAuthAccessToken: session.accessToken,
+          };
+          sendRequest(bodyToSend);
+        }
+        break;
+      case 'organisation':
+        allValidated = checkIfEmpty([firstName, lastName, country, nameOfOrg]);
+        if (allValidated && !loading && session) {
+          bodyToSend = {
+            type: 'organization',
+            firstname: firstName,
+            lastname: lastName,
+            name: nameOfOrg,
+            country: country,
+            mayPublish: !isPrivateAccount,
+            mayContact: isSubscribed,
+            oAuthAccessToken: session.accessToken,
+          };
+          sendRequest(bodyToSend);
+        }
+        break;
+      default:
+        setSnackbarMessage('Some Error has occured');
+        setSeverity("error")
+        handleSnackbarOpen();
+        break;
+    }
+  };
+
+  const SelectType = (type: any) => {
+    let name;
+    switch (type) {
+      case 'individual':
+        name = 'Individual';
+        break;
+      case 'RO':
+        name = 'Reforestation Organisation';
+        break;
+      case 'education':
+        name = 'School';
+        break;
+      case 'organisation':
+        name = 'Company';
+        break;
+      default:
+        name = 'Reforestation Organisation';
+        break;
+    }
+    return name;
+  };
+
+  if (loading || ( !loading && session && session.userExistsInDB) || (!loading && !session)) {
+    return null;
+  }
+
+  return (
+    <div
+      className={styles.signUpPage}
+      style={{
+        backgroundImage: `url(${process.env.CDN_URL}/media/images/app/bg_layer.jpg)`,
+        display:'flex',
+        flexDirection:'column'
+      }}
+    >
+      <div className={requestSent ? styles.signupRequestSent : styles.signup }>
+        {/* header */}
+      <div className={styles.header}>
+        <div
+          onClick={() => signOut({ callbackUrl: '/' })}
+          className={styles.headerBackIcon}
+        >
+          <BackArrow color={styles.primaryFontColor} />
+        </div>
+        <div className={styles.headerTitle}>Complete Signup</div>
+      </div>
+      {/* type of account buttons */}
+        <div className={styles.btnContainer}>
+          <button
+            type="button"
+            className={
+              accountType === 'individual' ? styles.btnColor : styles.btnSize
+            }
+            onClick={() => setAccountType('individual')}
+          >
+            <p
+              className={
+                accountType === 'individual'
+                  ? styles.accountTypeTextSelected
+                  : styles.accountTypeText
+              }
+            >
+              Individual{' '}
```

Comment:
I assume this patch has no translations at all at this time. I propose doing software development with translations from the very beginning, so they cannot be forgotten. That’s how I experienced it in my other projects.